genestack_client/study.py

```diff
@@ -0,0 +1,88 @@
+# coding=utf-8
+from genestack_client import Application
+
+
+class StudyDesign(Application):
+    """
+    API for working with studies and administrating the Study Design application.
+
+    Study Design is an application for describing studies and samples using Genestack metainfo
+    mechanism.
+    """
+    APPLICATION_ID = 'genestack/study-design'
+
+    def get_study_info(self, study_accession):
+        """
+        Get information about the given study from the project management point of view.
+
+        :param study_accession: study accession
+        :type study_accession: str
+        :return: dictionary with the following entries:
+
+            * ``studyNumber``: number of study unique for the organization,
+            * ``studyStatus``: current workflow step,
+            * ``approved``: mark of passed curator's review,
+            * ``hasApprovalButton``: mark of *Approve/Disapprove* button visibility in UI,
+            * ``approvalButtonHidden``: *true* if *Approve/Disapprove* button was hidden via
```

Comment:
```
hasApprovalButton
approvalButtonHidden
``` 
But we don't have any button in python client.

I suggest to rename this entries. Or maybe replace them with one 3-state (yes, no, hidden)

Reply:
> But we don't have any button in python client.

But we do in UI (as the docstring states)

> I suggest to rename this entries

This is not possible, this API is already being used by client scripts through direct invokes